server.py

```diff
@@ -121,7 +123,11 @@ def my_portal():
     activity = _db.images.find({"user_uid": session.get("logged_in")}).sort('created', pymongo.DESCENDING).limit(5)
     activity = [x for x in activity]
     return render_template("my_portal.html", domains=allowed_domains, total=total, images=activity)
-
+@app.route('/i/<filename>')
```

Comment:
that should be done with nginx or storage provider

Reply:
Can you give an example?